src/main/java/seedu/address/logic/commands/LocateCommand.java

```diff
@@ -31,6 +31,7 @@
             + "\n\t"
             + "Example:\t\t" + COMMAND_WORD + " -n alice bob charlie";
 
+    public static final String MESSAGE_NO_PERSON = "No one can be located!";
```

Comment:
This might be a little vague! Perhaps this could be rephrased to something like "Locate Command unsuccessful: No such person with those keyword(s) found!"